### PR TITLE
[RDY] vim-patch:7.4.576 

### PR DESCRIPTION
**Pull request by hemmer**
Problem:    Redrawing problem with 'relativenumber' and 'linebreak'.
Solution:   Temporarily reset 'linebreak' and restore it in more places.
        (Christian Brabandt)

https://github.com/vim/vim/commit/v7-4-576

(This is my first PR on github so apologies if I've made any mistakes!)

### CODE REVIEW

**Comment by hemmer** [comment]
See also issue #1946.

File: src/nvim/normal.c [hunk 1]
```diff
@@ -1434,8 +1434,11 @@ void do_pending_operator(cmdarg_T *cap, int old_col, bool gui_yank)
              || oap->op_type == OP_COLON
              || oap->op_type == OP_FUNCTION
              || oap->op_type == OP_FILTER)
-            && oap->motion_force == NUL)
+            && oap->motion_force == NUL) {
+          // make sure redrawing is correct
```
Comment (glts) [inline src/nvim/normal.c]: Style: comments capitalised and with punctuation `Make ... correct.`

File: src/nvim/normal.c [hunk 2]
```diff
@@ -1657,8 +1672,11 @@ void do_pending_operator(cmdarg_T *cap, int old_col, bool gui_yank)
       if (empty_region_error) {
         vim_beep();
         CancelRedo();
-      } else
-        op_replace(oap, cap->nchar);
+      } else {
+          // Restore linebreak, so that when the user edits it looks as before.
+          curwin->w_p_lbr = lbr_saved;
+          op_replace(oap, cap->nchar);
```
Comment (glts) [inline src/nvim/normal.c]: Indent with two spaces. (Have you seen our style guide? http://neovim.io/develop/style-guide.xml)

**Comment by glts** [comment]
Please remove `#1946` from the commit message subject, it's misleading as it points to an issue other than this pull request.

If you are confident that this patch does fix issue #1946, you can say `Closes #1946.` in your commit message body. Then that issue will automatically be closed when this pull request gets merged. Check the log for examples.

Other than that, this looks good and works as advertised. Good job! :+1:

**Comment by justinmk** [comment]
Merged, thanks @hemmer

**Pull request closed without merging**